R/pev_parameters.R

```diff
@@ -88,6 +90,8 @@ set_pev_epi <- function(
   min_wait,
   booster_timestep,
   booster_coverage,
+  booster_timed_coverage = NULL,
```

Comment:
I wonder if this can be simplified given that boosters will always be associated with the primary serires.

Would the following reduced arguments work:
`timesteps` define the vaccine coverage change timepoints at which primary series coverage or booster coverage changes.
`booster_coverage` (like, `coverages`) is a vector of length timesteps of booster coverages
Boosters a given at `booster_timestep` post primary series. For clarity this could be renamed (`booster_lags` or `booster_spacing`?)

That would remove the requirement for the two addtional arguments here:   `booster_timed_coverage`  `booster_timed_coverage_timestep`

Reply:
@giovannic @lmhaile 
Any thoughts on if the above would be sensible/helpful?
Happy to re-review otherwise

Reply:
Hi Pete, thanks for boosting! Yes I agree that` booster_timed_coverage_timestep` could be removed in favor of `timesteps`, as boosters will always be associated with the primary series. 

Also agree that `booster_timestep` could be renamed to `booster_spacing` to avoid confusion.

We may need to keep `booster_coverage` and `booster_timed_coverage` as is to avoid breaking the code of people implementing flat booster coverage across time.

Otherwise the changes look great-- everything is working as expected now!

Reply:
Hey! Sorry for the delay!

I think I get it and it makes a lot of sense. I can get on it later this week.

Reply:
Just to confirm, `booster_coverage` would need to be a matrix? Because we have (potentially) several boosters and several timesteps.

Reply:
Sorry for the delay while I was OOO Giovanni! Yes, `booster_coverage` would need to be a matrix to account for more than one booster.

Hope you had a nice new year and just let me know if you have other thoughts!